src/Typeahead.jsx

```diff
@@ -67,6 +74,18 @@ class Typeahead extends React.Component {
     });
   }
 
+  componentWillUnmount() {
+    if (this.props.scrollParentClass) {
+      window.removeEventListener('resize', this.hide)
+    }
+  }
+
+  hide() {
```

Comment:
Since we're using `this.state.hide` I think we should name this something different like `handleHide` otherwise it might be confusing to see `this.state.hide` next to `this.hide`